project/Dependencies.scala

```diff
@@ -21,7 +21,8 @@ object Dependencies {
     "com.google.cloud" % "google-cloud-compute" % "0.118.0-alpha",
     "org.broadinstitute.dsde.workbench" %% "workbench-google2" % workbenchGoogle2,
     "org.broadinstitute.dsde.workbench" %% "workbench-google2" % workbenchGoogle2 % Test classifier "tests",
-    "org.scalatestplus" %% "scalacheck-1-14" % "3.2.2.0" % Test
+    "org.scalatestplus" %% "scalacheck-1-14" % "3.2.2.0" % Test,
+    "org.scalatestplus" %% "mockito-3-3" % "3.2.0.0" % Test // https://github.com/scalatest/scalatestplus-selenium
```

Comment:
I don't think its possible to create an instance of the google `Bucket` class without interacting with an API to get/create, so I needed to mock it for testing